backend/drm/renderer.c

```diff
@@ -380,9 +380,15 @@ static struct wlr_drm_fb *drm_fb_create(struct wlr_drm_backend *drm,
 
 	if (formats && !wlr_drm_format_set_has(formats, attribs.format,
 			attribs.modifier)) {
+		const struct wlr_pixel_format_info *info =
+			drm_get_pixel_format_info(attribs.format);
+		if (!info) {
+			wlr_log(WLR_ERROR, "drm format info not found");
```

Comment:
Nit: can we maybe use a more descriptive error message (like "Failed to fallback on DRM opaque substitute format" that you've used above)? And also log the format code in hex?